front/app/modules/commercial/id_id_card_lookup/messages.ts

```diff
@@ -37,4 +37,8 @@ export default defineMessages({
     id: 'app.modules.id_id_card_lookup.cancel',
     defaultMessage: 'Cancel',
   },
+  helpImageAltText: {
+    id: 'app.modules.id_id_card_lookup.helpImageAltText',
+    defaultMessage: 'Help',
```

Comment:
Here we can also add a more descriptive alt text. This isn't telling what can be seen on the picture. I think it can be similar to the COW one.

Reply:
Added! tnx